src/backend/app.js

```diff
@@ -19,16 +20,69 @@ app.use(express.json());
 
 app.use(cors());
 
-router.use("/meals", mealsRouter);
+router.use('/meals', mealsRouter);
+
+const errMessage = 'Sorry, there are no meals.';
+
+app.get('/future-meals', async (req, res) => {
+  const [row] = await knex.raw(
```

Comment:
Can use just a variable to assign data.

Reply:
Sorry, I don't understand, what you mean. Can you explain, please?

Reply:
> Instead of "[rows]", "rows" is also fine.